gateway-service/src/main/java/org/zowe/apiml/gateway/security/service/ServiceAuthenticationServiceImpl.java

```diff
@@ -123,6 +124,9 @@ public AuthenticationCommand getAuthenticationCommand(String serviceId, String j
             if (found == null) {
                 // this is the first record
                 found = auth;
+
+                RequestContext.getCurrentContext().set(AUTHENTICATION_COMMAND_KEY, new ServiceAuthenticationServiceImpl.UniversalAuthenticationCommand());
+                RequestContextUtils.setInstanceInfo(instance);
```

Comment:
If there is a problem with the request, it should be handled in the AuthenticationCommand's apply and applyToRequest methods.